examples/dashboard-operator/config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/jrjohnson-gke/dashboard-operator:latest
+      - image: /dashboard-operator:latest
```

Comment:
This is a tricky problem - I agree we should merge as-is until we find a better approach!